player.html

```diff
@@ -1,130 +1,154 @@
-                </div>
-            </a>
+                <hr class="separator">
+                <button class="comment-button" type="button"><a href="">Comment</a></button>
+                <button class="comment-button" type="button"><a href="">Cancel</a></button>
+            </section>
 
-            <a href="">
-                <div class="suggest9">
-                    <img src="images/9. Networking.jpg" height="200" width="300" alt="Networking">
-                    <p class="suggest-head9"><b>Networking Essentials</b></p>
-                    <p class="desc9">Learn networking and merge that into your professional career development <b>Read More</b></p>
-                </div>
-            </a>
+            <section class="sidebar">
+                <h6>Suggested Videos...</h6>
 
-            <a href="">
-                <div class="suggest10">
-                    <img src="images/10. Cable Handling.jpg" height="200" width="300" alt="Network Cable">
-                    <p class="suggest-head10"><b>Cable Handling</b></p>
-                    <p class="desc10">Learn the proper way to handle messy network cables to make your work easier <b>Read More</b></p>
-                </div>
-            </a>
-        </div>
-
-        <!--
-        <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/AO_PNnBETJI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
-            -->
-        <iframe class="playlist" width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL2mBBjjiD8PmH-qb2S4rwhxKOl4j6uyfG" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
-        
-        <div class="follow-container">
-            <h4>Tag Attributes, adding Favicon and creating Lists</h4>
-            <button class="letter" type="button"><a href=""><b>L</b></a></button>
-            <h3 class="extra"><a href="">+ 9 more videos in the playlist</a></h3>
-            
-            
-            <h3 class="social-handle"><a href="https://www.youtube.com/@anyarsencarta" target="_blank">Learn I. T. with Encarta</a></h3>
-            <a class="subs"><b>256k subscribers</b></a>
-            <button class="subscribe" type="button"><a href=""><b>SUBSCRIBE</b></a></button>
-            <button class="share" type="button"><a href=""><b>SHARE</b></a></button>
-            
-            <div class="description">
-            <p><b>16 views  Dec 31, 2022</b> <br> In this tutorial, we show you more on Tag Attributes, how to add a favicon and how to create lists in html.</p>
-            </div>
-        </div> 
-
-        <div class="commenter">
-            <img src="images/My Pic.jpg" height="200" width="300" alt="Picture">
```

Comment:
Please consider changing the buttons in your project, since you are not creating forms. 🚀 Using simple <a> links could work great just as the buttons could :+1: